src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -120,7 +120,7 @@ public sealed class Arguments : TransformInputBase
             public TextNormKind VectorNormalizer = TextNormKind.L2;
         }
 
-        public sealed class Settings
+        public sealed class Options
```

Comment:
Can you add a summary tag for each of these settings, and for the Options class? You can take them from the constructor I believe.

Reply:
fixed

---
In reply to: [253319985](https://github.com/dotnet/machinelearning/pull/2394#discussion_r253319985) [](ancestors = 253319985)